scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -58,14 +85,39 @@ case class ExplicitReturnTypes(mirror: Mirror)
         typ <- defnType(defn)
       } yield ctx.addRight(replace, s": ${typ.treeSyntax}")
     }.to[Seq]
+
+    import ctx.config.explicitReturnTypes._
+
+    def matchesMemberVisibility(mods: Seq[Mod]): Boolean =
+      memberVisibility.contains(visibility(mods))
+
+    def matchesMemberKind(defn: Defn): Boolean =
+      kind(defn).exists(memberKind.contains)
+
+    def matchesSimpleDefinition(rhs: Term): Boolean =
+      rhs.is[Lit] && skipSimpleDefinitions
```

Comment:
With borrowing @olafurpg 's comment, I would define a "simple definition" as "its right-hand-side is a plain literal".

https://github.com/scalacenter/scalafix/pull/211/files/d49d74bbbb7eeece82e70af0df6f8b3ecf76d3f4#r121351776

I would also think a return type of a plain literal looks obvious, and needn't to be rewritten. What do you think about it?